src/API/Management/Jobs.php

```diff
@@ -59,6 +59,40 @@ public function importUsers($file_path, $connection_id, $params = [])
         return $request->call();
     }
 
+
+    /**
+     * Create an export users job.
+     * Required scope: "read:users"
+     *
+     * @param  array  $params
+     * @return mixed
+     */
+    public function exportUsers($params = [])
+    {
+        $request = $this->apiClient->method('post')
+        ->addPath('jobs', 'users-exports');
+
+        $body = [];
+
+        if (!empty($params['connection_id'])) {
+            $body['connection_id'] = filter_var($params['connection_id'], FILTER_SANITIZE_STRING);
```

Comment:
Do we typically use `filter_var` in our validation? When should/shouldn't we? Also curious about the `FILTER_SANITIZE_STRING`, if that's something we should be doing in other cases to, or any practical implications.

Reply:
It's _probably_ not necessary in this case, to be fair. I'll remove it for the time being. It offers some XSS mitigation and is something I want to investigate integrating where appropriate in the library, but I need to do more research on that first.